ui/tournamentSchedule/css/_side.scss

```diff
@@ -40,6 +41,13 @@
 
 .tour__links {
   margin-top: 1em;
+  a {
+	  color: $c-font;
+	  @include transition();
+	  &:hover {
+	  	color: $c-link;
+  }
```

Comment:
![](https://external-preview.redd.it/EcwKGjcxeR77frd8nFeG2ggEpCSLtGbudsABDPGBTe8.png?auto=webp&s=738e71260d3ec2e1f8ec863ad3f1de9b6aff51ad)